admin-dev/themes/new-theme/js/app/utils/growl.js

```diff
@@ -22,11 +22,12 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-export function showGrowl(type, message) {
+export function showGrowl(type, message, duration) {
+  duration = typeof(duration) != "undefined" ? duration : 2000;
```

Comment:
It also work like that, and don't use `"` for string
```suggestion
  duration = duration != undefined ? duration : 2000;
```